garrysmod/lua/autorun/base_npcs.lua

```diff
@@ -87,31 +90,31 @@ AddNPC( {
 	Class = "npc_citizen",
 	Category = Category,
 	Model = "models/odessa.mdl",
-	KeyValues = { citizentype = CT_UNIQUE }
+	KeyValues = { citizentype = CT_UNIQUE, SquadName = "resistance" }
 }, "npc_odessa" )
 
 AddNPC( {
 	Name = "Medic",
 	Class = "npc_citizen",
 	Category = Category,
-	SpawnFlags = SF_CITIZEN_MEDIC,
-	KeyValues = { citizentype = CT_REBEL },
+	SpawnFlags = bit.bor( 8, 131072 ),

```

Comment:
I wrote this code about 5 months ago for a NPC enhancement mod, and I've always thought bit.bor only accepted integers, before I realized exactly what SF_FLAG_HERE meant and what kind of definition it was. I didn't think about it very much while making this pull request, except for wondering if there's another way to include multiple spawnflags.

I'll fix it as soon as possible.